apps/client/src/features/control/message/MessageControl.module.scss

```diff
@@ -13,6 +13,36 @@
   gap: $element-spacing;
 }
 
+.buttonSection {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, auto);
+  gap: $element-spacing;
+
+}
+
+.blinkingLabel {
+  grid-area: 1 / 1 / 2 / 2;
+  font-size: $inner-section-text-size;
+  color: $label-gray;
+}
+
+.blinkingButton {
+  grid-area: 2 / 1 / 3 / 2;
+}
+
+.blackoutLabel {
+  grid-area: 1 / 2 / 2 / 3;
+    font-size: $inner-section-text-size;
+  color: $label-gray;
+}
+
+.blackoutButton {
+  grid-area: 2 / 2 / 3 / 3;
+  width: 100%;
+  justify-self: end;
+}
+
```

Comment:
Mega nitpick list:

- Could you use grid-template-areas here. I find it so hard to think about this way of defining grid
- We should either a) decrease space between first and second row or b) increase space between timer stuff and public stuff. I am inclined towards a) as a better option
- mind indentation